conda/activate.py

```diff
@@ -409,9 +409,9 @@ def _prompt_modifier(self, conda_default_env):
         return "(%s) " % conda_default_env if context.changeps1 else ""
 
     def _get_activate_scripts(self, prefix):
-        return self.path_conversion(glob(join(
+        return self.path_conversion(sorted(glob(join(
             prefix, 'etc', 'conda', 'activate.d', '*' + self.script_extension
-        )))
+        ))))
```

Comment:
Let's keep all the logic in this helper method.  So prefer this over the change on line 173.

Reply:
Agreed and done.